### DISCUSSION TITLE
How to trigger a scene with mqtt?

### DISCUSSION THREAD

**Discussion by mdilg** [zwave-js/zwave-js-ui#2760] [Q&A] [answered]
I'm wondering how a scene that is configured in zwave-js-ui can be triggered by a mqtt message. What mqtt topic do I have to use? And what payload is expected?

I'm referring to this sentence in the Readme:

"Scene Management: Create scenes and trigger them by using MQTT apis (with timeout support)"

**Comment by robertsLando** [comment, marked as answer]
Check: https://zwave-js.github.io/zwave-js-ui/#/guide/mqtt?id=z-wave-apis

Specifically you should use `_activateScene` with a payload like:

```
{
  "args": [ xx ]
}
```

Topic: `<mqtt_prefix>/_CLIENTS/ZWAVE_GATEWAY-<mqtt_name>/api/_activateScene/set`

Just replace `xx` with your sceneID

**Reply by mdilg** [reply]
Thanks for pointing me to the part of the documentation where my question is answered:

=> https://zwave-js.github.io/zwave-js-ui/#/guide/mqtt?id=execute-scene

Did not see it before asking.

Works like a charm.

**Reply by robertsLando** [reply]
No worries :)
